examples/visco_elasticity/spring_maxwell_model.py

```diff
@@ -0,0 +1,167 @@
+                        [self.lam0, 2.0 * self.mu0 + self.lam0, self.lam0, 0.0, 0.0, 0.0],
+                        [self.lam0, self.lam0, 2.0 * self.mu0 + self.lam0, 0.0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, 2.0 * self.mu0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, 0.0, 2.0 * self.mu0, 0.0],
+                        [0.0, 0.0, 0.0, 0.0, 0.0, 2.0 * self.mu0],
+                    ]
+                )
+                self.D_1 = np.array(
+                    [
+                        [2.0 * self.mu1 + self.lam1, self.lam1, self.lam1, 0.0, 0.0, 0.0],
+                        [self.lam1, 2.0 * self.mu1 + self.lam1, self.lam1, 0.0, 0.0, 0.0],
+                        [self.lam1, self.lam1, 2.0 * self.mu0 + self.lam1, 0.0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, 2.0 * self.mu1, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, 0.0, 2.0 * self.mu1, 0.0],
+                        [0.0, 0.0, 0.0, 0.0, 0.0, 2.0 * self.mu1],
+                    ]
+                )
+
+            case Constraint.PLANE_STRAIN:
+                self.D_0 = np.array(
+                    [
+                        [2.0 * self.mu0 + self.lam0, self.lam0, self.lam0, 0.0],
+                        [self.lam0, 2.0 * self.mu0 + self.lam0, self.lam0, 0.0],
+                        [self.lam0, self.lam0, 2.0 * self.mu0 + self.lam0, 0.0],
+                        [0.0, 0.0, 0.0, 2.0 * self.mu0],
+                    ]
+                )
+                self.D_1 = np.array(
+                    [
+                        [2.0 * self.mu1 + self.lam1, self.lam1, self.lam1, 0.0],
+                        [self.lam1, 2.0 * self.mu1 + self.lam1, self.lam1, 0.0],
+                        [self.lam1, self.lam1, 2.0 * self.mu1 + self.lam1, 0.0],
+                        [0.0, 0.0, 0.0, 2.0 * self.mu1],
+                    ]
+                )
+
+            case Constraint.PLANE_STRESS:
+                self.D_0 = (
+                        self.E0
+                        / (1 - self.nu ** 2.0)
+                        * np.array(
+                    [
+                        [1.0, self.nu, 0.0, 0.0],
+                        [self.nu, 1.0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, (1.0 - self.nu)],
+                    ]
+                )
+                )
+                self.D_1 = (
+                        self.E1
+                        / (1 - self.nu ** 2.0)
+                        * np.array(
+                    [
+                        [1.0, self.nu, 0.0, 0.0],
+                        [self.nu, 1.0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, 0.0],
+                        [0.0, 0.0, 0.0, (1.0 - self.nu)],
+                    ]
+                )
+                )
+
+            case Constraint.UNIAXIAL_STRESS:
+                self.D_0 = np.array([[self.E0]])
+                self.D_1 = np.array([[self.E1]])
+            case _:
```

Comment:
Same question as before: Why no uniaxial strain? I do not really think that we need it, but it seems weird to exclude it if you considered all other cases

Reply:
see above